samples/package.json

```diff
@@ -17,7 +17,7 @@
   "dependencies": {
     "@google-cloud/error-reporting": "^0.6.0",
     "express": "^4.16.3",
-    "yargs": "^12.0.0"
+    "yargs": "^13.0.0"
```

Comment:
Does it make sense for this to be in dependencies rather than devDependencies?

Reply:
As samples are today, no. Each sample file is essentially a cli tool.